src/core/src/test/java/org/apache/jmeter/util/XPathUtilTest.java

```diff
@@ -276,4 +278,19 @@ public void testPutValuesForXPathInList() throws ParserConfigurationException, S
         assertEquals("one", matchs.get(0));
         assertEquals("two", matchs.get(1));
     }
+
+    @Test
```

Comment:
I will take merge test into our current test suite, as it passes with our current code, too :)